rmw_connextdds_common/src/common/rmw_impl.cpp

```diff
@@ -1530,6 +1579,123 @@ RMW_Connext_Subscriber::take_serialized(
   return rc;
 }
 
+rmw_ret_t
+RMW_Connext_Subscriber::set_cft_expression_parameters(
+  const char * filter_expression,
+  const rcutils_string_array_t * expression_parameters)
+{
+  RMW_CONNEXT_ASSERT(nullptr != filter_expression)
+  RMW_CONNEXT_ASSERT(!internal)
```

Comment:
Just to be safe, let's add an assertion for `RMW_CONNEXT_ASSERT(nullptr != node)`, just to keep track that that's a prerequisite too since `node` might be null in some cases. For now this only happens for `internal` endpoints, so the two assertions kind of overlap, but maybe in the future things might deviate and we might forget of the other prerequisite of this function, so the explicit reminder will help.

Reply:
updated